src/components/sidebar-area/sidebar-area.tsx

```diff
@@ -114,25 +123,33 @@ export function SidebarArea() {
                 }}
               >
                 <StatusCircle isConnected={isConnected} isLoading={isLoading} />
-                {hostname}
+                {isLoading ? (
+                  <LoaderCircle className='animate-spin' size={12} />
```

Comment:
Maybe you should use a skeleton component instead here, would look nicer than two circles